The job **`dist-x86_64-linux`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/12377807991/job/34548230759) [(plain)](https://github.com/rust-lang-ci/rust/commit/98b2d77e7808314ac4423bf802240f58e19cbf38/checks/34548230759/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
file:.git/config remote.origin.url=https://github.com/rust-lang-ci/rust
file:.git/config remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
file:.git/config gc.auto=0
file:.git/config http.https://github.com/.extraheader=AUTHORIZATION: basic ***
file:.git/config branch.try.remote=origin
file:.git/config branch.try.merge=refs/heads/try
file:.git/config remote.upstream.fetch=+refs/heads/*:refs/remotes/upstream/*
file:.git/config submodule.library/backtrace.active=true
file:.git/config submodule.library/backtrace.url=https://github.com/rust-lang/backtrace-rs.git
file:.git/config submodule.library/stdarch.active=true
---
   Compiling compiler_builtins v0.1.138
   Compiling profiler_builtins v0.0.0 (/checkout/library/profiler_builtins)
[RUSTC-TIMING] build_script_build test:false 0.111
[RUSTC-TIMING] build_script_build test:false 0.256
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: profiler_builtins@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
   Compiling rustc-std-workspace-core v1.99.0 (/checkout/library/rustc-std-workspace-core)
[RUSTC-TIMING] rustc_std_workspace_core test:false 0.024
   Compiling alloc v0.0.0 (/checkout/library/alloc)
   Compiling cfg-if v1.0.0
---
[RUSTC-TIMING] num_conv test:false 0.150
   Compiling time v0.3.37
[RUSTC-TIMING] indexmap test:false 1.345
   Compiling rustc_serialize v0.0.0 (/checkout/compiler/rustc_serialize)
warning: tikv-jemalloc-sys@0.6.0+5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
error: failed to run custom build command for `tikv-jemalloc-sys v0.6.0+5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7`
note: To improve backtraces for build dependencies, set the CARGO_PROFILE_RELEASE_BUILD_OVERRIDE_DEBUG=true environment variable to enable debug information generation.
Caused by:
  process didn't exit successfully: `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/release/build/tikv-jemalloc-sys-e9174cf3cf19f895/build-script-build` (exit status: 101)
  --- stdout
  TARGET=x86_64-unknown-linux-gnu
  TARGET=x86_64-unknown-linux-gnu
  HOST=x86_64-unknown-linux-gnu
  NUM_JOBS=16
  OUT_DIR="/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out"
  BUILD_DIR="/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build"
  SRC_DIR="/cargo/registry/src/index.crates.io-6f17d22bba15001f/tikv-jemalloc-sys-0.6.0+5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7"
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_JEMALLOC_OVERRIDE
  cargo:rerun-if-env-changed=JEMALLOC_OVERRIDE
  OPT_LEVEL = Some(3)
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out)
  TARGET = Some(x86_64-unknown-linux-gnu)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CC_x86_64-unknown-linux-gnu
  CC_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CC_x86_64_unknown_linux_gnu
  CC_x86_64_unknown_linux_gnu = Some(sccache clang)
  cargo:rerun-if-env-changed=CC_KNOWN_WRAPPER_CUSTOM
  CC_KNOWN_WRAPPER_CUSTOM = None
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some(false)
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  CFLAGS_x86_64-unknown-linux-gnu = None
  CFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CFLAGS_x86_64_unknown_linux_gnu
  CFLAGS_x86_64_unknown_linux_gnu = Some(-ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  CARGO_ENCODED_RUSTFLAGS = Some(--cfg=windows_raw_dylib-Csymbol-mangling-version=v0-Zunstable-options--check-cfg=cfg(bootstrap)--check-cfg=cfg(llvm_enzyme)-Zmacro-backtrace-Csplit-debuginfo=off-Wrustc::internal-Drustc::symbol_intern_string_literal-Wkeyword_idents_2024-Wunsafe_op_in_unsafe_fn-Clink-args=-Wl,-z,origin-Clink-args=-Wl,-rpath,$ORIGIN/../lib-Clink-arg=-fuse-ld=lld-Zon-broken-pipe=kill-Zdefault-visibility=protected-Zdylib-lto-Clto=thin-Cembed-bitcode=yes-Clink-args=-Wl,--icf=all-Cprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo-Cllvm-args=-vp-counters-per-site=4-Cllvm-args=-static-func-strip-dirname-prefix=2)
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out)
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  TARGET = Some(x86_64-unknown-linux-gnu)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  CFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CFLAGS_x86_64_unknown_linux_gnu
  CFLAGS_x86_64_unknown_linux_gnu = Some(-ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out)
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  TARGET = Some(x86_64-unknown-linux-gnu)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  CFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CFLAGS_x86_64_unknown_linux_gnu
  CFLAGS_x86_64_unknown_linux_gnu = Some(-ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out)
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  TARGET = Some(x86_64-unknown-linux-gnu)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
  CFLAGS_x86_64-unknown-linux-gnu = None
  cargo:rerun-if-env-changed=CFLAGS_x86_64_unknown_linux_gnu
  CFLAGS_x86_64_unknown_linux_gnu = Some(-ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  cargo:warning=Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
  CC="clang"
  CFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin"
  JEMALLOC_REPO_DIR="jemalloc"
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_JEMALLOC_SYS_WITH_MALLOC_CONF
  cargo:rerun-if-env-changed=JEMALLOC_SYS_WITH_MALLOC_CONF
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_JEMALLOC_SYS_WITH_LG_PAGE
  cargo:rerun-if-env-changed=JEMALLOC_SYS_WITH_LG_PAGE
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_JEMALLOC_SYS_WITH_LG_HUGEPAGE
  cargo:rerun-if-env-changed=JEMALLOC_SYS_WITH_LG_HUGEPAGE
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_JEMALLOC_SYS_WITH_LG_QUANTUM
  cargo:rerun-if-env-changed=JEMALLOC_SYS_WITH_LG_QUANTUM
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_JEMALLOC_SYS_WITH_LG_VADDR
  cargo:rerun-if-env-changed=JEMALLOC_SYS_WITH_LG_VADDR
  CARGO_FEATURE_STATS not set
  running: cd "/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build" && CC="clang" CFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin" CPPFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin" LDFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin" "sh" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build/configure" "--with-version=5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7" "--disable-cxx" "--enable-doc=no" "--enable-shared=no" "--with-private-namespace=_rjem_" "--disable-stats" "--host=x86_64-unknown-linux-gnu" "--build=x86_64-unknown-linux-gnu" "--prefix=/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out"
  checking for x86_64-unknown-linux-gnu-gcc... clang
  checking whether the C compiler works... no
  checking whether the C compiler works... no
  running: "tail" "-n" "100" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build/config.log"
  XSLROOT=''
  XSLTPROC='false'
  a=''
  abi=''
  abs_objroot='/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build/'
  abs_srcroot='/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build/'
  ac_ct_CC=''
  ac_ct_CXX=''
  bindir='${exec_prefix}/bin'
  build='x86_64-unknown-linux-gnu'
  build_alias='x86_64-unknown-linux-gnu'
  build_cpu=''
  build_os=''
  build_vendor=''
  cfghdrs_in=''
  cfghdrs_out=''
  cfgoutputs_in=''
  cfgoutputs_out=''
  datadir='${datarootdir}'
  datarootdir='${prefix}/share'
  docdir='${datarootdir}/doc/${PACKAGE}'
  dvidir='${docdir}'
  enable_autogen=''
  enable_cache_oblivious=''
  enable_cxx='no'
  enable_debug=''
  enable_doc='no'
  enable_experimental_smallocx=''
  enable_fill=''
  enable_initial_exec_tls=''
  enable_lazy_lock=''
  enable_log=''
  enable_opt_safety_checks=''
  enable_opt_size_checks=''
  enable_prof=''
  enable_readlinkat=''
  enable_shared='no'
  enable_static=''
  enable_stats='no'
  enable_tls=''
  enable_uaf_detection=''
  enable_utrace=''
  enable_xmalloc=''
  enable_zone_allocator=''
  exe=''
  exec_prefix='/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out'
  host='x86_64-unknown-linux-gnu'
  host_alias='x86_64-unknown-linux-gnu'
  host_cpu=''
  host_os=''
  host_vendor=''
  htmldir='${docdir}'
  importlib=''
  includedir='${prefix}/include'
  infodir='${datarootdir}/info'
  install_suffix=''
  je_=''
  jemalloc_version=''
  jemalloc_version_bugfix=''
  jemalloc_version_gid=''
  jemalloc_version_major=''
  jemalloc_version_minor=''
  jemalloc_version_nrev=''
  libdir='${exec_prefix}/lib'
  libdl=''
  libexecdir='${exec_prefix}/libexec'
  libprefix=''
  link_whole_archive=''
  localedir='${datarootdir}/locale'
  localstatedir='${prefix}/var'
  mandir='${datarootdir}/man'
  o=''
  objroot=''
  oldincludedir='/usr/include'
  pdfdir='${docdir}'
  prefix='/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out'
  private_namespace=''
  program_transform_name='s,x,x,'
  psdir='${docdir}'
  rev='2'
  sbindir='${exec_prefix}/sbin'
  sharedstatedir='${prefix}/com'
  so=''
  srcroot=''
  sysconfdir='${prefix}/etc'
  target_alias=''

  ## ----------- ##
  ## confdefs.h. ##
  ## ----------- ##
  /* confdefs.h */
  #define PACKAGE_NAME ""
  #define PACKAGE_TARNAME ""
  #define PACKAGE_VERSION ""
---
  configure: error: in `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build':
  configure: error: C compiler cannot create executables
  See `config.log' for more details
  thread 'main' panicked at /rust/deps/tikv-jemalloc-sys-0.6.0+5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7/build.rs:388:9:
  command did not execute successfully: cd "/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build" && CC="clang" CFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin" CPPFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin" LDFLAGS="-O3 -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -ffunction-sections -fdata-sections -fPIC -m64 --target=x86_64-unknown-linux-gnu -fdebug-prefix-map=/checkout=/rustc/98b2d77e7808314ac4423bf802240f58e19cbf38 -fprofile-generate=/tmp/tmp-multistage/opt-artifacts/rustc-pgo -flto=thin" "sh" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out/build/configure" "--with-version=5.3.0-1-ge13ca993e8ccb9ba9847cc330696e02839f328f7" "--disable-cxx" "--enable-doc=no" "--enable-shared=no" "--with-private-namespace=_rjem_" "--disable-stats" "--host=x86_64-unknown-linux-gnu" "--build=x86_64-unknown-linux-gnu" "--prefix=/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-rustc/x86_64-unknown-linux-gnu/release/build/tikv-jemalloc-sys-b4726751c41b1815/out"
  stack backtrace:
     0:     0x5563a881d82a - std::backtrace_rs::backtrace::libunwind::trace::hb414cecefb265e28
                                 at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/library/std/src/../../backtrace/src/backtrace/libunwind.rs:116:5
     1:     0x5563a881d82a - std::backtrace_rs::backtrace::trace_unsynchronized::h13cd53bbd12d24e7
---
    17:     0x5563a878c88d - build_script_build::main::h5d66d0517ad2c598
    18:     0x5563a878eba3 - core::ops::function::FnOnce::call_once::hd23b4255ae2b02e7
    19:     0x5563a878e536 - std::sys::backtrace::__rust_begin_short_backtrace::haff39b806216b777
    20:     0x5563a878e4b9 - std::rt::lang_start::{{closure}}::h786c2453dc571beb
    21:     0x5563a8815c77 - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::hf2c13211747708f5
    22:     0x5563a8815c77 - std::panicking::try::do_call::h0c27488c576b542f
                                 at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/library/std/src/panicking.rs:573:40
    23:     0x5563a8815c77 - std::panicking::try::hf3c4e79d7f4e2d80
                                 at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/library/std/src/panicking.rs:536:19
---
[RUSTC-TIMING] aho_corasick test:false 8.868
[RUSTC-TIMING] regex_syntax test:false 8.895
[RUSTC-TIMING] time test:false 4.211
[RUSTC-TIMING] regex_automata test:false 13.028
warning: rustc_llvm@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: rustc_llvm@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: rustc_llvm@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: rustc_llvm@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: rustc_llvm@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
warning: rustc_llvm@0.0.0: Inherited flag "-fembed-bitcode=all" is not supported by the currently used CC
[2024-12-17T17:18:44.586Z INFO  opt_dist::timer] Section `Stage 1 (Rustc PGO) > Build PGO instrumented rustc and LLVM` ended: FAIL (490.90s)`
[2024-12-17T17:18:44.586Z INFO  opt_dist::timer] Section `Stage 1 (Rustc PGO)` ended: FAIL (490.90s)`
[2024-12-17T17:18:44.586Z INFO  opt_dist] Timer results
    -----------------------------------------------------------------
---
Caused by:
    Command RUST_BACKTRACE=full python3 /checkout/x.py build --target x86_64-unknown-linux-gnu --host x86_64-unknown-linux-gnu --stage 2 library/std --rust-profile-generate /tmp/tmp-multistage/opt-artifacts/rustc-pgo --set llvm.thin-lto=false --set llvm.link-shared=true [at /checkout/obj] has failed with exit code Some(1)

Stack backtrace:
   0: <anyhow::Error>::msg::<alloc::string::String>
             at /rust/deps/anyhow-1.0.94/src/backtrace.rs:27:14
   1: <opt_dist::exec::CmdBuilder>::run
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/exec.rs:80:17
   2: <opt_dist::exec::Bootstrap>::run
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/exec.rs:181:9
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/main.rs:225:13
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/main.rs:225:13
   4: <opt_dist::timer::TimerSection>::section::<opt_dist::execute_pipeline::{closure#1}::{closure#0}, ()>
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/timer.rs:111:22
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/main.rs:214:9
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/main.rs:214:9
   6: <opt_dist::timer::TimerSection>::section::<opt_dist::execute_pipeline::{closure#1}, opt_dist::training::RustcPGOProfile>
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/timer.rs:111:22
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/main.rs:211:29
   8: opt_dist::main
             at /rustc/98b2d77e7808314ac4423bf802240f58e19cbf38/src/tools/opt-dist/src/main.rs:401:18
   9: <fn() -> core::result::Result<(), anyhow::Error> as core::ops::function::FnOnce<()>>::call_once
   9: <fn() -> core::result::Result<(), anyhow::Error> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b4297a573b4eefacd62e7ea1ba071536282d3254/library/core/src/ops/function.rs:250:5
  10: std::sys::backtrace::__rust_begin_short_backtrace::<fn() -> core::result::Result<(), anyhow::Error>, core::result::Result<(), anyhow::Error>>
             at /rustc/b4297a573b4eefacd62e7ea1ba071536282d3254/library/std/src/sys/backtrace.rs:154:18
  11: std::rt::lang_start::<core::result::Result<(), anyhow::Error>>::{closure#0}
             at /rustc/b4297a573b4eefacd62e7ea1ba071536282d3254/library/std/src/rt.rs:195:18
  12: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
  13: std::panicking::try::do_call
             at /rustc/b4297a573b4eefacd62e7ea1ba071536282d3254/library/std/src/panicking.rs:557:40
  14: std::panicking::try
             at /rustc/b4297a573b4eefacd62e7ea1ba071536282d3254/library/std/src/panicking.rs:520:19
```

</details>

--- a/compiler/rustc_codegen_ssa/Cargo.toml
+++ b/compiler/rustc_codegen_ssa/Cargo.toml
@@ -8,7 +8,8 @@ edition = "2021"
 ar_archive_writer = "0.4.2"
 arrayvec = { version = "0.7", default-features = false }
 bitflags = "2.4.1"
-cc = "1.1.23"
+# Pinned so `cargo update` bumps don't cause breakage
+cc = "=1.2.0"
 either = "1.5.0"
 itertools = "0.12"
 pathdiff = "0.2.0"

--- a/library/profiler_builtins/Cargo.toml
+++ b/library/profiler_builtins/Cargo.toml
@@ -11,4 +11,5 @@ doc = false
 [dependencies]
 
 [build-dependencies]
-cc = "1.2"
+# Pinned so `cargo update` bumps don't cause breakage
+cc = "=1.2.0"

